mira/terarium_client.py

```diff
@@ -0,0 +1,93 @@
+"""Client functionality to Terarium."""
+
+from typing import Dict, List, Union
+
+import jsonschema
+import requests
+from metamodel import TemplateModel
+from pydantic import BaseModel
+
+from mira.modeling import Model
+from mira.modeling.askenet.petrinet import AskeNetPetriNetModel
+
+__all__ = [
+    "associate",
+    "post_template_model",
+    "post_amr",
+    "post_amr_remote",
+]
+
+
+def associate(*, project_id: str, model_id: str) -> str:
+    """Associate a model (UUID) to a project (UUID) and return the association UUID."""
+    x = f"http://data-service.staging.terarium.ai/projects/{project_id}/assets/models/{model_id}"
+    res = requests.post(x)
+    return res.json()["id"]
+
+
+def sanity_check_amr(amr_json):
+    assert "schema" in amr_json
+    schema_json = requests.get(amr_json["schema"]).json()
+    jsonschema.validate(schema_json, amr_json)
```

Comment:
We could put this in `mira.sources.askenet` (it will apply to any AMR) and the other sanitize function I implemented into `mira.metamodel.utils`?

Reply:
Template model sanity check added in e18c25c.
Moved AMR sanity check in 9469f52